Sources UI updates in docs (#824)
rudderlabs/rudderstack-docs: docs/sources/cloud-apps/webhook-source.mdx

---
title: "Webhook Source"
description: Step-by-step guide on adding a Webhook source in RudderStack.
---

RudderStack lets you add any source that supports a webhook and use it to send events to your preferred destinations.

<div class="warningBlock">
You can configure webhooks as sources only for the platforms that support them.
</div> 

## Getting started

Follow these steps to set up your webhook source in the RudderStack dashboard:

<div class="infoBlock">

> As an example, events are ingested from <a href="https://mailchimp.com">Mailchimp</a> into RudderStack by configuring a webhook.

Reviewer: ```suggestion
In this example, events are ingested from <a href="https://mailchimp.com">Mailchimp</a> into RudderStack by configuring a webhook.
```
